packages/frontend/src/components/Dialog/CloseFooterAction.tsx

```diff
@@ -13,7 +13,9 @@ export default function CloseFooterAction({ onClose }: DialogProps) {
   return (
     <DialogFooter>
       <FooterActions>
-        <FooterActionButton onClick={onClose}>{tx('close')}</FooterActionButton>
+        <FooterActionButton styling='primary' onClick={onClose}>
+          {tx('close')}
+        </FooterActionButton>
```

Comment:
IDK about this. It's not the primary action e.g. in the "new chat" dialog

![image](https://github.com/user-attachments/assets/ba72e8f3-5d07-4d22-bc5f-f093fd8ab671)

Reply:
So the question is: is a single button always the "primary" action or is there no idea of "primary" - "secondary" at all.

Reply:
I'm sure it's specified somewhere in the Material guidelines (not to say that this is the law of design, but If everyone follows it, we should be too probably)

Reply:
In this particular case, I think a close button at the top would make more sense.
But perhaps this is for a separate merge request.